package.json

```diff
@@ -3,7 +3,6 @@
   "version": "1.0.0-rtm.4",
   "description": "A keyboard driver for cycle.js",
   "main": "dist/index.js",
-  "typings": "dist/index.d.ts",
   "scripts": {
```

Comment:
Um, not deprecated. "typings" in package.json is still used, maybe it can be updated to "types" which is synonymous with "typings" anyway. Refer https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html